### PR TITLE
Dev backend

### PR DESCRIPTION
**Pull request by LeRatierBretonnien**
One fix (ie nbPlayers in /credit command) and 1 enhancement (ie allow rollMode settings when using "bypass" mode)

### CODE REVIEW

File: scripts/market-wfrp4e.js [hunk 1]
```diff
@@ -436,7 +428,7 @@ class MarketWfrp4e {
         } else //generate a card with a summary and a receive button
         {
             let amount
-            let nbActivePlayers = getNbOfActivePlayers();
+            let nbActivePlayers = game.users.players.filter(p => p.data.active).length;
```
Comment (moo-man) [inline scripts/market-wfrp4e.js]: How does this fix anything? It's the same line of code.

Comment (LeRatierBretonnien) [reply scripts/market-wfrp4e.js]: Update.... This is the proper fix ...

File: scripts/utility-wfrp4e.js [hunk 2]
```diff
@@ -636,9 +636,10 @@ class WFRP_Utility
     {
       let percentage = 0;
       let matchCounter = 0;
-      for (let i = 0; i < key.length; i++)
+      let myword = object[key].toLowerCase();
```
Comment (moo-man) [inline scripts/utility-wfrp4e.js:640]: If this is lowercase, shouldn't query be as well?

Comment (LeRatierBretonnien) [reply scripts/utility-wfrp4e.js:640]: Yes, but in fact in the code the upper functions are doing it.
This why i "lowercased" it. But for more security, i am adding it to the query also in the function itself.

**Pull request merged**